### ISSUE TITLE
Please add this to your wiki/readme - regarding java.lang.UnsatisfiedLinkError: no jlibtorrent in java.library.path

### ISSUE THREAD

**Issue by quantumcat1** [frostwire/frostwire-jlibtorrent#153]
I kept getting java.lang.UnsatisfiedLinkError: no jlibtorrent in java.library.path no matter what version I chose, even though I read in previous issues that it was "fixed".

I figured out how to fix it (I am using eclipse neon on Windows 8). I am making this issue so future Googlers can find it. It took a lot of time, effort, and headache to work out so I want to let this effort help someone else.

First which you probably already did, is download the jar from https://mvnrepository.com/artifact/com.frostwire/jlibtorrent/1.2.0.6-RC1 and add it by going to project properties->build path->libraries->new external jar-> and select this jar

Open a command window, cd to the project you're trying to use this jar in, and run:
wget --no-check-certificate -P bin/release/windows/x86 https://s3.amazonaws.com/jlibtorrent1/release/windows/x86/jlibtorrent.dll

Then

wget --no-check-certificate -P bin/release/windows/x86_64 https://s3.amazonaws.com/jlibtorrent1/release/windows/x86_64/jlibtorrent.dll

Then, go to project properties->build path->libraries and look for jlibtorrent 1.2.0.6. Expand the little arrow icon on the left. 

Click "native library location" then the Edit button to the right. Go to your project folder, then the bin folder, then release, then choose either x86 or 64 (if you have 32bit or 64bit version of windows). And click OK/Apply

Now try and run your code. It will magically work!!!!!

**Comment by aldenml** [comment]
Well, this solution is not correct, since those urls point to my latest builds and will probably crash your run (also that's my private S3 bucket, and it's not free).

You better download the binaries from maven corresponding to the same version and unzip it to copy the binary.

**Comment by quantumcat1** [comment]
Thanks, where do I find them? I can't find anything by searching the web and https://github.com/frostwire/frostwire-jlibtorrent/tree/master/binaries no longer exists

**Comment by aldenml** [comment]
http://search.maven.org/#search%7Cga%7C1%7Cjlibtorrent

**Comment by quantumcat1** [comment]
I downloaded the windows jar, and tried setting the jlibtorrent jar's native library directory to the folder containing it, and it didn't work. Only the dll will work, as downloaded through the script :-/
Someone is testing my program, who has Windows 10 64 bit, and he gets this exception, although it works fine on my Windows 8 64 bit computer. I tried compiling a version with the x86 binary, just in case, but that didn't work for him either. I don't suppose you have any idea why that could be? Or is there a script that can be run to find out which binary is required by a machine? If not, that's OK. I read an issue raised about 1.5 years ago where it looks possible to specify what path to use - so I'll try and learn about that.
Thanks for making this library by the way, it is super helpful!!!
